docks/op_expand.go

```diff
@@ -40,7 +41,9 @@ type ExpandOp struct {
 	// recvProxy is populated with the configured recv function
 	recvProxy func() <-chan *container.Container
 	// sendProxy is populated with the configured send function
-	sendProxy func(c *container.Container) *terminal.Error
+	sendProxy func(c *container.Container, highPriority bool) *terminal.Error
+	// prioMsgs holds the number of messages to forward with high priority.
+	prioMsgs *int32
 }
```

Comment:
why using a pointer for that? that allocates additional memory for the pointer and the storage. better just inline the counter

Reply:
Atomic access.